src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,87 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
-
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
-
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button
-          data-cy="clearSearchButton"
-          type="button"
-          className="delete"
+import { Status } from '../../types/Status';
+
+type Props = {
+  query: string;
+  setFilter: (value: Status) => void;
+  setQuery: (value: string) => void;
+};
+
+const SELECT_VALUES = [Status.all, Status.active, Status.completed];
```

Comment:
you may use `Object.values(Status)`. That should also allow you to get rid of the typecasting below, in the `handleFilterChange`